### PR TITLE
MINOR: Updated the package name to correct one.

### PR DESCRIPTION
**Pull request by kamalcph**
### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by kamalcph** [comment]
How to resolve this issue in IntelliJ? I've invalidated the IntelliJ cache and built it again but couldn't resolve this one.

Screenshot 2021-02-10 at 9 54 09 PM

**Comment by chia7712** [comment]
@kamalcph Thanks for your patch.

1. [ControllerContextTest](https://github.com/apache/kafka/blob/trunk/core/src/test/scala/unit/kafka/controller/ControllerContextTest.scala) has similar issue.
1. the [config](https://github.com/apache/kafka/blob/trunk/core/src/test/scala/kafka/tools/DefaultMessageFormatterTest.scala#L144) need to be updated from `unit.kafka.tools` to `kafka.tools`

**Comment by kamalcph** [comment]
@chia7712 
Addressed your review comments. Please take another look.

File: checkstyle/import-control-core.xml [hunk 1]
```diff
@@ -65,7 +65,6 @@
     <allow pkg="kafka.api"/>
     <allow pkg="kafka.server"/>
     <allow pkg="org.apache.kafka.clients.admin"/>
-    <allow pkg="integration.kafka.server" class="IntegrationTestHelper"/>
```
Comment (chia7712) [inline checkstyle/import-control-core.xml:68]: It seems the class `IntegrationTestHelper` was moved due to review comment (#9986)

this should be included by separate PR and the `core/src/test/java/kafka/test/junit/README.md` need to be updated as well.


@mumrah FYI

File: core/src/test/scala/integration/kafka/server/IntegrationTestUtils.scala [hunk 2]
```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-package integration.kafka.server
+package kafka.server
```
Comment (chia7712) [inline core/src/test/scala/integration/kafka/server/IntegrationTestUtils.scala:18]: I feel this package name is intentional. @mumrah WDYY?

File: checkstyle/import-control-core.xml (hunk 1 above)
Comment (kamalcph) [reply checkstyle/import-control-core.xml:68]: Ack. Opened a new PR #10126.

File: core/src/test/scala/integration/kafka/server/IntegrationTestUtils.scala (hunk 2 above)
Comment (kamalcph) [reply core/src/test/scala/integration/kafka/server/IntegrationTestUtils.scala:18]: @mumrah Can you take a look?
